### DISCUSSION TITLE
[Bug report] Theme override for scripts.js not generating scripts.min.js

### DISCUSSION THREAD

**Discussion by bttrx** [GetPublii/Publii#1893] [Bug Report]
### Operating system

Linux Mint 22 MATE

### Publii version

0.46.2 (build 16966)

### Issue type

Application

### Bug description

I created a theme override for 'Terminal'. It didn't work for the first time, because I missed to place `scripts.min.js` to `terminal-override/assets/js/`.

I would expect Publii generating `scripts.min.js` on-the-fly from my customized `terminal-override/assets/js/scripts.js`.

**Comment by bobmitro** [comment]
It's not a bug; you can create the .min.js file yourself.

**Reply by bttrx** [reply]
Um, okay, maybe not a bug, but a feature request then. Of course, I can create the .min.js myself, but `style.css` is already created automatically for the `terminal-override/assets/css/` folder. So, why not `scripts.min.js` too?
